app/Database/CourseInsertion.hs

```diff
@@ -19,8 +19,9 @@ import qualified Data.ByteString.Lazy.Char8 as BSL
 import Happstack.Server.SimpleHTTP
 import Control.Monad.IO.Class (MonadIO)
 import Control.Monad.Trans.Reader (ReaderT)
-import Data.Maybe (fromMaybe)
+import Data.Maybe (fromMaybe) 
 import Config (databasePath)
+import Database.Persist --Kael
```

Comment:
Always explicitly state the functions/types which are being imported, in parentheses after the module name. (The other import statements do this.)